services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/RouteOptionsTest.java

```diff
@@ -384,16 +395,17 @@ public void baseUrlWithLastSlash() {
 
   @Test
   public void routeOptionsWithDecodedChars_toUrlWithEncodedChars() {
-    String expectedEncodedUrl = "https://api.mapbox.com/directions/v5/mapbox/driving/-122.4003312,37.7736941;-122.4187529,37.7689715?access_token=pk.token&geometries=polyline6&waypoint_names=my%20starting%20position;my%20destination";
+    String expectedEncodedUrl =
+        "https://api.mapbox.com/directions/v5/mapbox/driving/-122.4003312,37.7736941;-122.4187529,37.7689715?access_token=pk.token&geometries=polyline6&waypoint_names=my%20starting%20position;my%20destination";
     List<Point> coordinates = new ArrayList<>();
     coordinates.add(Point.fromLngLat(-122.4003312, 37.7736941));
     coordinates.add(Point.fromLngLat(-122.4187529, 37.7689715));
 
     RouteOptions options = RouteOptions.builder()
-      .profile(DirectionsCriteria.PROFILE_DRIVING)
-      .coordinatesList(coordinates)
-      .waypointNames("my starting position;my destination")
-      .build();
+        .profile(DirectionsCriteria.PROFILE_DRIVING)
```

Comment:
sometimes we use 2 and sometimes 4 spaces. Which option is correct, I will reformat all the file.

Reply:
Historically `mapbox-java` follows the 4 spaces Java formatting and https://github.com/mapbox/mapbox-java/tree/main/config/checkstyle as opposed to the Kotlin 2 spaces code style we have in https://github.com/mapbox/mapbox-navigation-android. I'd rather to stick with the 4 spaces for consistency.

Reply:
Thank you. Now I understand where the inconsistency comes from. My Android Studio uses settings from Mapbox Navigation Android. 

As I see the majority of code uses 2 as indent and continuation indent. So to have less changes in PR for easier review I stick to 2 indent size formatting.

Reply:
> As I see the majority of code uses 2 as indent and continuation indent. So to have less changes in PR for easier review I stick to 2 indent size formatting.

I'd thought it was the other way around, but yeah ultimately not a big deal 👍